docs/toc.yml

```diff
@@ -345,7 +345,9 @@
         href: excel/excel-add-ins-worksheets.md
         displayName: Excel
     - name: Excel JavaScript API overview
-      href: https://docs.microsoft.com/office/dev/add-ins/reference/overview/excel-add-ins-reference-overview
+
+
+href: https://docs.microsoft.com/office/dev/add-ins/reference/overview/excel-add-ins-reference-overview
```

Comment:
Should this be a relative link?

Reply:
You know I'm not sure - I'm not sure why it isn't relative which makes me think it is there on purpose?

Reply:
This might have been a relic from when these overview pages were in the -reference repo. Don't worry about it with this PR. I'm sure Kim's TOC rework will correct this.